crossdata-server/src/main/resources/server-reference.conf

```diff
@@ -0,0 +1,173 @@
+crossdata-server.akka.cluster.seed-node-timeout = 5s
```

Comment:
Clean default-value parameters. We should have only specific properties and parameters which we change frequently